test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -261,6 +270,8 @@ private static void validateSpec(RestSpec restSpec) {
 
     @After
     public void wipeCluster() throws Exception {
+        // Running tasks might create indices and otherwise invalidate the wipe. Lets just let the finish.
+        waitForAllRunningTasksToFinish();
```

Comment:
Hrm, shouldn't deleting an index cause any tasks for that index to be cancelled, internally? Seems this would be something a user could hit, and that we simply did not correctly cleanup all resources when deleting an index?